horace_core/sqw/@data_sqw_dnd/get_projection.m

```diff
@@ -9,24 +9,39 @@
 %
 alatt=obj.alatt;
 angdeg=obj.angdeg;
```

Comment:
surprised these are in the data object, should be in sample

Reply:
this is the mess needing the rearrangement